elide-datastore/elide-datastore-jpa/pom.xml

```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>com.yahoo.elide</groupId>
             <artifactId>elide-datastore-multiplex</artifactId>
-            <version>4.4.5-SNAPSHOT</version>
+            <version>RELEASE</version>
```

Comment:
This needs to remain as a SNAPSHOT dependency for maven release to work correctly.